frontend/src/components/shared/RepoClone.vue

```diff
@@ -238,6 +238,7 @@
     enableFinetune: Boolean,
     enableEvaluation: Boolean,
     showAddToCollections: Boolean,
+    syncStatus: String
```

Comment:
We may don't need this attr, as we already have 'repo' passed in, and the 'repo' here is repoDetailStore actually